chapters/VK_EXT_debug_utils.txt

```diff
@@ -44,6 +44,12 @@ include::../api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
 include::../validity/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
+.Valid Usage
+****
+  * pname:pNameInfo\->pname:objectType must: not be ename:VK_OBJECT_TYPE_UNKNOWN
+  * pname:pNameInfo\->pname:objectHandle must: not be dlink:VK_NULL_HANDLE
+****
```

Comment:
This VU block is in the wrong place. Should be above Implicit VU.

Reply:
:+1: